apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nav-sidebar/src/style.scss

```diff
@@ -0,0 +1,128 @@
+@import '~@wordpress/base-styles/colors';
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+
+$sidebarWidth: 272px;
+$border: 1px solid $light-gray-500;
+$transitionPeriod: 200ms;
+
+.is-wpcom-block-editor-nav-sidebar-attached {
+	.edit-post-layout {
+		transition: margin-left $transitionPeriod;
+		@include reduce-motion( 'transition' );
+	}
+
+	.edit-post-fullscreen-mode-close {
+		transition: width $transitionPeriod;
+		@include reduce-motion( 'transition' );
+	}
+}
+
+.is-wpcom-block-editor-nav-sidebar-opened {
+	.edit-post-layout {
+		margin-left: $sidebarWidth;
+	}
+
+	.edit-post-fullscreen-mode-close {
+		width: 0;
+	}
+}
+
+.is-wpcom-block-editor-nav-sidebar-close-hidden {
+	.edit-post-fullscreen-mode-close {
+		opacity: 0;
+		padding: 0;
+		pointer-events: none;
+	}
+}
+
+.wpcom-block-editor-nav-sidebar__container {
+	position: fixed;
+	top: 0;
+	left: -$sidebarWidth;
+	bottom: 0;
+	width: $sidebarWidth;
+	border-right: $border;
```

Comment:
It looks like this border is currently visible when the sidebar is closed, looking like a 1px white line next to the W button. I think it might be because the border isn't included in the sidebarWidth, so the `left: -$sidebarWidth` might need the border width added to it.

Here's how the W button looks when the sidebar is closed in Chrome:

![image](https://user-images.githubusercontent.com/14988353/82530031-a9320180-9b7f-11ea-80ff-5d7f07ddd904.png)

And when it's open:

![image](https://user-images.githubusercontent.com/14988353/82530043-b2bb6980-9b7f-11ea-9686-b24a769453cb.png)

Reply:
Good spotting, and thanks for tracking down where that problem was too! I prefer `box-sizing: border-box;` which is probably why I always write code assuming it's already applied to everything 😄 I've switched the sidebar to use it.

Reply:
Interestingly, wp-admin/Gutenberg should already use border-box:

https://github.com/WordPress/gutenberg/blob/8ce6ad15013665896de8f37a3de73795227ceb8d/packages/base-styles/_mixins.scss#L245-L252

Reply:
Maybe reset of the reset CSS isn't applied either in the sidebar?